projects/storefrontlib/src/shared/services/promotion/promotion.module.ts

```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
+import { PromotionService } from './promotion.service';
+
+@NgModule({
+  providers: [PromotionService],
```

Comment:
Using 
```
@Injectable({
  providedIn: 'root',
})
```
on service would make this unnecessary (and improve treeshakeability) .